provisioning/azure/scripts/deploy.sh

```diff
@@ -22,11 +22,20 @@ kubectl apply -f ./provisioning/kubernetes/deploy/templates-api.yaml
 kubectl apply -f ./provisioning/kubernetes/deploy/azure/service.yaml
 kubectl rollout status deployment/templates-api --timeout=900s
 
-TEMPLATES_API_IP=$(kubectl get services \
-  --selector "app=templates-api" \
-  --namespace templates-api \
-  --no-headers \
-  --output jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+TEMPLATES_API_IP=""
+TIME_WAITED=0
+# every 10 seconds but in total max 15 minutes try to fetch TEMPLATES_API_IP
+#shellcheck disable=2203
+while [[ -z "$TEMPLATES_API_IP" && $TIME_WAITED -lt 15*60 ]]; do
+    echo "Waiting for Templates API ingress IP..."
+    sleep 10;
+    TIME_WAITED=$((TIME_WAITED + 10))
+    TEMPLATES_API_IP=$(kubectl get services \
+      --selector "app=templates-api" \
+      --namespace templates-api \
+      --no-headers \
+      --output jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+done
```

Comment:
@kacurez pridal som cakanie na IP pre azure

Reply:
BTW: nastavil si `TEMPLATES_API_ETCD_REPLICAS=1` pre vsetky stacks, alebo to mam (niekde) spravit ja?

Reply:
nenastavil som, to prosim urob:
- pre azure stacky staci rucne pridat do varaible groups pre jednotlive stacky https://dev.azure.com/keboola-prod/Keboola%20Connection%20Azure/_library?itemType=VariableGroups
- pre aws stacky treba cez PR v https://github.com/keboola/kbc-stacks

Reply:
@kacurez  na ten azure nemam opravnenia, aby som to zmenil :/

Reply:
ok, do azure stackov som vsade nastavil `TEMPLATES_API_ETCD_REPLICAS=1`

Reply:
PR do Aws: https://github.com/keboola/kbc-stacks/pull/290